src/README.md

```diff
@@ -0,0 +1,25 @@
+## Overview

```

Comment:
We can use the main Readme

Reply:
It uses html in the file. I dont think you will be able to use it.

Reply:
because has to work in visual studio and other readers